src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue

```diff
@@ -1,54 +1,54 @@
 <template>
 	<div class="faq">
-		<h2 class="faq__header">
+		<h2 class="tw-mb-2">
 			Frequently asked questions
 		</h2>
 		<!-- eslint-disable max-len -->
 		<ul class="faq__list">
 			<li>
-				<h4>Are my contributions loans or donations?</h4>
+				<h3>Are my contributions loans or donations?</h3>
```

Comment:
@ryan-ludwig @roger-in-kiva  is this the same way we've handled FAQs on the other MG page?

Reply:
No, most FAQs are powered by Contentful and use `<kv-frequently-asked-questions>`